### PR TITLE
chore: open source UI

### PR DESCRIPTION
**Pull request by Krithika3**
Signed-off-by: Krithika Vijayakumar <krithika_vijayakumar@intuit.com>

### CODE REVIEW

**Comment by Krithika3** [comment]
This PR just moves the UI directory over

File: ui/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Prerequisites
+
+Follow the instructions below to set up your development environment.
+
+### Install requirements
+
+These tools are required for development.
+
+1. [`NumaFlow`](https://github.com/numaproj/numaflow/blob/master/docs/DEVELOPMENT.md): Follow the document to prepare `Dataflow` development environment.
```
Comment (whynowy) [inline ui/README.md:25]: `Dataflow`?

File: ui/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+1. [`node`](https://nodejs.org/en/download/): NodeJS - `brew install node`.
+1. [`yarn`](https://yarnpkg.com/): Package manager - `brew install yarn`
+
+### Create a dev cluster
```
Comment (whynowy) [inline ui/README.md:29]: Maybe we don't need all the stuffs below, they should be covered in the root development doc.

Comment (Krithika3) [reply ui/README.md]: Removed

File: ui/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # NumaFlow UI
 
-![Numaflow Image](../docs/images/Numa.svg)
+![Numaflow Image](../docs/architecture/assets/numa.svg)
```
Comment (whynowy) [inline ui/README.md:3]: Not under `architecture` but directly under `docs`. The ones under `architecture` should also move over later.

Comment (Krithika3) [reply ui/README.md]: Done. should we move the others too?

Comment (vigith) [reply ui/README.md]: let's not move the architecture yet. We can keep the "Improvement Proposals" and the related architecture separate.

**Review by vigith — APPROVED** [review]

**Comment by vigith** [comment]
I have a question about the UX code structure. Why not keep wrappers along with the main code and avoid utils dir altogether?

**Comment by Krithika3** [comment]
> I have a question about the UX code structure. Why not keep wrappers along with the main code and avoid utils dir altogether?

I would like to discuss this with our UI team before we do it. Merging this for now

**Pull request merged**